<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>examination</class>
 <widget class="QMainWindow" name="examination">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模拟测试工具V1.0 作者：shiyu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pB_examination_stop">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>考试结束</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>题目总数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_examination_sum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>80</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>当前题号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_examination_CurrentQid">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>80</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>考试科目：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>80</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>考生姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>80</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>考生身份证号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_examinstion_subject">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>200</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_examinstion_name">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>200</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_examinstion_num">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>200</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>80</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>考试类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_examinstion_type">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>200</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_examination_last">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一题</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_examination_next">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一题</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>考试计时：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_examination_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_examination_ExamineAnswer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查看当前题答案</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_examination_ExamineAnswer">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>题目栏：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_examinati_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>500</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>答案栏：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_A">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_B">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_C">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_D">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_abandon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>放弃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>80</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>当前题库题号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_examination_CurrentQid_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_answer_A">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>500</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_answer_B">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>500</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_answer_C">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>500</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_answer_D">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>500</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
